builtin/v8/manifest/manifest.go

```diff
@@ -0,0 +1,111 @@
+package manifest
```

Comment:
Maybe this should be outside the v8 structure?
We dont anticipate it changing, amd it is kind of self versioned.